flexmeasures/ui/crud/assets.py

```diff
@@ -267,8 +269,15 @@ def owned_by(self, account_id: str):
             user_can_create_assets=user_can_create_assets(),
         )
 
+    @use_kwargs(
+        {
+            "start_time": AwareDateTimeField(format="iso", required=False),
```

Comment:
Good that you found our preferred way of validation!

(If an error happens, the result is not that pretty though, it's more meant for API usage)

Anyway, my other test case was if just one tjem is given. Then the page is not applying the other, even. Maybe we should simply insist that either both or neither of them are passed, before we render the view.

Reply:
We should also check if start_time < ennd_time.